sbom/sbom.mql.yaml

```diff
@@ -6,6 +6,7 @@ packs:
     name: Mondoo SBOM
     filters:
       - mql: asset.family.contains("unix")
+      - mql: asset.family.contains('windows')
```

Comment:
Multiple entries should be and connected not OR, so this should not work

Reply:
Then, we have a bug in the filters. See comment on the kernel query.